.travis.yml

```diff
@@ -8,12 +8,15 @@ sudo: false
 # http://docs.travis-ci.com/user/caching/
 cache: pip
 
-
 env:
-  - SPHINXOPTS="-W"
+  - SPHINXOPTS="-W" SPHINX_VERSION=1.2.3
+  - SPHINXOPTS="-W" SPHINX_VERSION=latest
 
 before_install:
-  - pip install Sphinx==1.2.3
+  - if [[ $SPHINX_VERSION == '1.2.3' ]]; then pip install Sphinx==1.2.3; fi
+  - if [[ $SPHINX_VERSION == 'latest' ]]; then pip install Sphinx; fi
   - pip install docutils==0.12
 
-script: make clean html
+script:
+  - if [[ $SPHINX_VERSION == '1.2.3' ]]; then make clean html; fi
```

Comment:
What's the reason for not using `make clean html` for all versions?  Also, as above `!= 'latest'` would also apply here.

Reply:
Primarily because `make clean html` will build both documentation sets and that the OMERO documentation requires more work to build with Sphinx 1.6.